File: spologp_v/lista2sala-C/questao6.c
#include<stdio.h>
#include<math.h>

int main()
{
    float valorA, valorB, valorC, delta, x1, x2, x, raiz, raiz2;

    printf("Digite o valor A:");
    scanf("%f", &valorA);
    printf("Digite o valor B:");
    scanf("%f", &valorB);
    printf("Digite o valor C:");
    scanf("%f", &valorC);

    delta = pow(-valorB, 2)-4*valorA*valorC;

if(delta>0){
    raiz = (int)sqrt(delta);
    x1 = (-1)*-valorB+raiz/(2*valorA);
    x2 = (-1)*-valorB-raiz/(2*valorA);
    printf("Esse valor possui duas raízes reais e distintas: %.1f, %.1f", x1, x2);
}

else if(delta==0){
    raiz2 = (int)sqrt(delta);
    x = -valorB+raiz2/(2*valorA);
    printf("Esse número possui apenas uma raiz real: %.1f", x);
}

else{
    printf("Esse número não possui raízes reais.");
}

return 0;
}

File: spologp_v/lista2sala-python/questao6.py
A = float(input("Digite o número A:"))
B = float(input("Digite o número B:"))
C = float(input("Digite o número C:"))

delta = (-B)**2-4*A*C

if(delta>0):
    x1 = float
    x2 = float
    x1 = (-1)*(-B)+delta**0.5/(2*A)
    x2 = (-1)*(-B)-delta**0.5/(2*A)
    print("Essa equação possui duas raízes reais e distintas:", '%.2f' %x1, '%.2f' %x2)
elif(delta==0):
    x = float
    x = -B+delta**(1/2)/(2*A)
    print("Esse número possui raízes reais e iguais:", '%.2f' %x)
else:
    print("Esse número não possui raízes reais.")